src/actions.rs

```diff
@@ -533,7 +534,29 @@ pub fn update_template(
                 .context("perform template cache")?;
             Ok(true)
         }
-        TemplateComparison::Changed | TemplateComparison::TargetNotRegularFile => {
+        TemplateComparison::Changed => {
+            // At this point, we're not sure if there's a difference between the rendered source
+            // and target, only that the target has been modified in some way.
+            let diff = generate_template_diff(source, target, handlebars, variables, false)
+                .context("diff source and target")?;
+            if diff_nonempty(&diff) {
+                error!(
+                    "Updating template {:?} -> {:?} but {}. Skipping",
+                    source, target.target, comparison
+                );
+                if log_enabled!(log::Level::Info) {
+                    info!("Refusing because of the following changes in source: ");
```

Comment:
`in source` -> `in target location`

LGTM otherwise 🎉

Reply:
Thank brain fog, should be good to go now!